crawl-ref/source/player.cc

```diff
@@ -3167,6 +3179,13 @@ int player_stealth()
             stealth += STEALTH_PIP;
     }
 
+    //Faerie Dragons' bright wings reduce stealth.
+    if (you.species == SP_FAERIE_DRAGON && (you.form == transformation::none
+        || you.form == transformation::appendage))
```

Comment:
Did you forget blade hands?

Reply:
Great catch, I did. And dragon form. Fixed for both.